src/pages/learn/schema.mdx

```diff
@@ -254,30 +290,28 @@ type Droid implements Character {
 }
 ```
 
-You can see that both of these types have all of the fields from the `Character` interface, but also bring in extra fields, `totalCredits`, `starships` and `primaryFunction`, that are specific to that particular type of character.
-
-Interfaces are useful when you want to return an object or set of objects, but those might be of several different types.
+You can see that both of these types have all of the fields from the `Character` Interface type, but also bring in extra fields—`totalCredits`, `starships` and `primaryFunction`—that are specific to that particular type of character.
 
-For example, note that the following query produces an error:
+Interface types are useful when you want to return an object or set of objects, but those might be of several different types. For example, note that the following query produces an error:
 
 ```graphql
-# { "graphiql": true, "variables": { "ep": "JEDI" } }
-query HeroForEpisode($ep: Episode!) {
-  hero(episode: $ep) {
+# { "graphiql": true }
+{
+  hero(episode: JEDI) {
```

Comment:
My thinking here was that because #1802 swaps the order of this page in the left nav so that it comes before the operation-related content that it might be confusing to use the variable syntax here until that concept is introduced on the next page.

Reply:
I'm okay with that; thank you for sharing the reasoning :+1: